ui/js/ui.transfers.js

```diff
@@ -38,11 +38,17 @@ var UI = (function(UI, $, undefined) {
         if (!rawAmount) {
           throw UI.t("amount_cannot_be_zero");
         } else {
+          if(rawAmount < 0){
+            throw UI.t("amount_cannot_be_negative");
+          }
           amount = iota.utils.convertUnits(parseFloat(rawAmount), rawUnits, "i");
 
           if (!amount) {
             throw UI.t("amount_cannot_be_zero");
           }
+          if(amount < 0){
+            throw UI.t("amount_cannot_be_negative");
+          }
```

Comment:
If `rawAmount` is positive, then `amount` will be always > 0, I don't think this check is needed, the first one is enough.

Reply:
That's a good point. I guess I had it twice since they checked for both `rawAmount` = 0 and `amount` = 0